pkg/flowaggregator/clickhouseclient/clickhouseclient.go

```diff
@@ -0,0 +1,511 @@
+// Copyright 2021 Antrea Authors
```

Comment:
```suggestion
// Copyright 2022 Antrea Authors
```
It should be the year of first publication of the file